src/stac_utils/ngpvan.py

```diff
@@ -151,3 +151,84 @@ def get_paginated_items(self, url, **kwargs):
             next_full_url = data.get("next_page_link")
             next_url = next_full_url.split("/")[-1] if next_full_url else None
         return all_items
+
+    def format_person_json(self, row: dict, id_key: str, has_identifier: bool) -> dict:
+        formatted_json = {
+            "firstName": row.get("first_name"),
+            "lastName": row.get("last_name"),
+            "dateOfBirth": row.get("date_of_birth"),
+            "contactMode": "Person",
+        }
+
+        if has_identifier and id_key is not None:
+            formatted_json["identifiers"] = [
+                {"type": "votervanid", "externalId": row.get(id_key)}
+            ]
+        elif has_identifier and id_key is None:
+            raise ValueError("did not indicate name of id key column")
+        elif row.get("custom_field_id") and row.get("custom_field_group_id"):
+            formatted_json["customFieldValues"] = [
+                {
+                    "custom_field_id": row.get("custom_field_id"),
+                    "custom_field_group_id": row.get("custom_field_group_id"),
+                    "assignedValue": row.get(id_key),
+                }
+            ]
+        else:
+            print("No ID key used")
+
+        if row.get("email"):
+            formatted_json["emails"] = [{"email": row.get("email")}]
+
+        if row.get("phone"):
+            formatted_json["phones"] = [
+                {"phoneNumber": str(row.get("phone")).replace(".0", "")}
+            ]
+
+        if row.get("middle_name"):
+            formatted_json["middleName"] = row.get("middle_name")
+
+        # if row.get('street_address') or row.get('city') or row.get('stateOrProvince') or row.get('zipOrPostalCode'):
+        address = {}
+
+        if row.get("street_address"):
+            address["addressLine1"] = row.get("street_address")
+
+        if row.get("city"):
+            address["city"] = row.get("city")
+
+        if row.get("state") or row.get("stateOrProvince"):
+            if row.get("state"):
+                address["stateOrProvince"] = row.get("state")
+            else:
+                address["stateOrProvince"] = row.get("stateOrProvince")
+
+        if row.get("zip") or row.get("zipOrPostalCode"):
+            if row.get("zip"):
+                address["zipOrPostalCode"] = row.get("zip")
+            else:
+                address["zipOrPostalCode"] = row.get("zipOrPostalCode")
+
+        if address is not {}:
+            formatted_json["addresses"] = [address]
+
+        return strip_dict(formatted_json)
+
+    def van_phone_validation(self, phone: str) -> str:
```

Comment:
let's rename this to "validate_phone" or something to maintain the verb-led convention

Reply:
done